co2eq/purchase/footprints.yml

```diff
@@ -1104,6 +1104,16 @@ _children:
             intensityKilograms: 0.893
             source: 'https://iopscience.iop.org/1748-9326/13/4/044004/media/ERL_044004_SD.pdf'
             unit: kg
+            conversion:
+              portion:
+                kilograms: 0.060
+                incrementStepSize: 1
+                source: 'https://www.yazio.com/en/foods/lychee.html'
+                default: true
+              lychee:
```

Comment:
shouldn't it be a `unit`?

Reply:
wouldn't we have 1 lychee instead of 1 unit in the app

Reply:
could be, but for consistency of the data structure I would keep "unit" or "item" (we should use the same as the `unit` we use for buying one washing machine for example)

Reply:
Could we then say that if something is called unit then in the app we show the parent's name instead?

Reply:
yep we could!
I just checked and we use `unit: item` for items, so the correct conversion should be `item`.
This will allow us to automatically convert between units.